cellfinder_napari/train.py

```diff
@@ -183,7 +184,7 @@ def widget(
         if Model_weights == Path.home():
             Model_weights = None
 
-        if YAML_files[0] == Path.home():
+        if YAML_files[0] == Path.home():  # type: ignore
```

Comment:
I can fix this in #56, but for now this will pass `mypy`

Reply:
Cool!